Project architecture/Mobile app architecture.md

```diff
@@ -166,7 +168,7 @@ class MachineFriendlyNameRepository @Inject constructor(
 
 ```
 
-Let's first take a look at the injected dependencies. As you may have noticed, the only components that we use in the repository are interactors and mappers. `MachineFriendlyNameRepository` has a very clear purpose, it provides a way to apply simple CRUD operations on the domain model. In this case, read and update (named *fetch* and *modify* to separate repository naming from standard API call naming). To achieve this, the repository knows which interactors can be used to extract the needed data. `getStoredMachineName` is an ideal interactor, because it can fetch a String from SharedPreferences which we can propagate  to the repository's fetch method, which is used by a UseCase that needs the name from local storage. But, having an ideal interactor is rarely the case, so let's focus on `getMachineInfo` interactor. As the name says, it doesn't just fetch the machine name, but a source-specific data model `MachineInfo`. Since `MachineFriendlyNameRepository` works with a String as the domain model, we need a way to create a machine friendly name from `MachineInfo`. For that purpose, we use `machineInfoToMachineNameMapper`. Keep in mind, mappers can become significally more complex than this and therefore, they need to be thoroughly tested!
+Let's first take a look at the injected dependencies. As you may have noticed, the only components that we use in the repository are interactors and mappers. `MachineFriendlyNameRepository` has a very clear purpose, it provides a way to apply simple CRUD operations on the domain model. In this case, read and update (named *fetch* and *modify* to separate repository naming from standard API call naming). To achieve this, the repository knows which interactors can be used to extract the needed data. `getStoredMachineName` is an ideal interactor, because it can fetch a String from SharedPreferences which we can propagate  to the repository's fetch method, which is used by a UseCase that needs the name from local storage. But, having an ideal interactor is rarely the case, so let's focus on `getMachineInfo` interactor. As the name says, it doesn't just fetch the machine name, but a source-specific data model `MachineInfo`. Since `MachineFriendlyNameRepository` works with a String as the domain model, we need a way to create a machine friendly name from `MachineInfo`. For that purpose, we use `machineInfoToMachineNameMapper`. Keep in mind, mappers can become significantly more complex than this and therefore, they need to be thoroughly tested!
```

Comment:
A few minor points here.
1. I think this section would be more readable and easier to keep reader's attention if it had some line breaks, as it is a longer chunk of text. 
2. We mention how repository only has interactors and mappers here. This might not always be true, but the main purpose of repository should always stay the same. For example: In my project I use classes called `Converters` to map stuff from domain layer to the source layer. I would go more abstract than to name classes here and instead describe the purpose of a repository -> In best case scenario it aggregates all classes enabling CRUD operations for one domain model. In some cases maybe 2. This means that it may contain interactors, mappers, converters etc. But that set might get bigger. And the class naming might change especially with the new mappers as extensions. The main point though, should stay the same, so I would personally rather focus on explaining the point of the repository using classes here as examples, but pointing out that those are not concrete.

What do you think?

Reply:
1. Done ✅ 
2. I added a sentence or two clearing this up. I used what you mentioned about the converters as an example of how the repository implementation could differ. 😁 

In this PR, I didn’t really do any major changes in the Mobile app architecture chapter since it was already well-written and still very applicable to today’s architecture. 🚀